lib/linter/apply-disable-directives.js

```diff
@@ -124,7 +122,7 @@ module.exports = ({ directives, problems, reportUnusedDisableDirectives = "off"
         .map(directive => Object.assign({}, directive, { unprocessedDirective: directive }))
         .sort(compareLocations);
 
-    const lineDirectives = lodash.flatMap(directives, directive => {
+    const lineDirectives = directives.flatMap(directive => {
```

Comment:
Per [node.green](https://node.green/), `Array.prototype.flatMap` is available on Node >= 12.4, while ESLint 7 supports these versions:

https://github.com/eslint/eslint/blob/ebfb63a682004a008f2707dbad616e5ae1630b2c/package.json#L144-L146

so this change would have to wait for the next ESLint major release.